docs/development.rst

```diff
@@ -1122,127 +1460,177 @@ to him until directed).
 Adding a DFE model
 ******************
 
```

Comment:
I modified the DFE section of the docs to mirror previous sections (adding species and demographic models). This probably should be reviewed by whomever wrote this initially, to make sure it's still accurate.

Reply:
The rewrite looks fine to be. We can give it another look before selection paper/v0.2.1, too.